### PR TITLE
Removal of CSRF token from GET requests #75

### PR DESCRIPTION
**Pull request by szendo**

### CODE REVIEW

**Comment by dumbNickname** [comment]
Looks good, but maybe it would be possible to add a test here?

Maybe we could also add CSRF token for 'delete' requests: http://stackoverflow.com/questions/11833061/is-csrf-possible-with-put-or-delete-methods? Disclaimer: I do not know how is it currently supported by oasp4j and what are spring security defaults when it comes to delete requests.

**Comment by szendo** [comment]
Added CSRF for DELETE :)

File: app/main/layout/navbar.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
         </div>
         <div class="collapse navbar-collapse" data-ng-class="{'in':!navCollapsed}">
             <ul class="ng-cloak nav navbar-nav" data-ng-show="currentUser.isLoggedIn()">
-                <li><a href="#table-mgmt/table-search" translate>OASP.TABLES</a></li>
+                <li><a href="#table-mgmt/table-search" data-ng-show="currentUser.getUserName() === 'Willy Waiter'" translate>OASP.TABLES</a></li>
```
Comment (dumbNickname) [inline app/main/layout/navbar.html]: Do we want a hard-coded user name here? Shouldn't it be based on a user role?

Comment (hohwille) [inline app/main/layout/navbar.html]: Good finding. This has to be changed/fixed.

Comment (szendo) [inline app/main/layout/navbar.html]: I did like the next line ( Cook and Cheif. ).
"data-ng-show="currentUser.getUserName() === 'Carl Cook' || currentUser.getUserName() === 'Charly Chief'"

We must change all of them, everywhere. But we need new issue.

**Pull request merged**